includes/ucf-news-config.php

```diff
@@ -12,6 +12,7 @@ class UCF_News_Config {
 				'topics'    => '',
 				'limit'     => 3,
 				'per_row'   => 3,
+				'show_img'  => true,
```

Comment:
Could we use `show_image` instead of `show_img` for consistency with the Post List SC plugin? https://github.com/UCF/UCF-Post-List-Shortcode#usage

Reply:
Updated~~